### PR TITLE
Etcm 8880 gov init in wizard

### PR DESCRIPTION
**Pull request by jankun4**
# Description

Adds run_init_governance step in `prepare-configuration` wizard step.
[link](https://input-output.atlassian.net/browse/ETCM-8880)

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages.
- [ ] New tests are added if needed and existing tests are updated.
- [ ] Relevant logging and metrics added
- [ ] CI passes. See note on CI.
- [ ] Any changes are noted in the `changelog.md` for affected crate
- [ ] Self-reviewed the diff

# Note on CI
If your PR is from a fork, the necessary CI jobs won't trigger automatically for security reasons.
You will need to get someone with write privileges.  Please contact IOG Partner Chains developers to do this
for you.

### CODE REVIEW

File: toolkit/offchain/src/init_governance/mod.rs [hunk 1]
```diff
@@ -12,11 +12,39 @@ mod tests;
...
+	async fn init_governance(
+		&self,
+		governance_authority: MainchainAddressHash,
+		payment_key: MainchainPrivateKey,
+		genesis_utxo_id: UtxoId,
+	) -> Result<OgmiosTx, OffchainError> {
+		run_init_governance(governance_authority, payment_key, Some(genesis_utxo_id), self)
+			.await
+			.map_err(|_| {
+				OffchainError::InternalError("Init governance failed".into())
```
Comment (LGLO) [inline toolkit/offchain/src/init_governance/mod.rs:38]: Can we display cause error?

**Review by LGLO — APPROVED** [review]
Please format it `cargo fmt` and print also reason of failure

**Review by AmbientTea — APPROVED** [review]
run it locally with the rest of the flow. it works.

File: toolkit/partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs [hunk 2]
```diff
@@ -17,6 +20,15 @@ pub fn prepare_main_chain_config<C: IOContext>(
...
 	cardano_parameteres.save(context);
 	set_up_cardano_addresses(context, genesis_utxo, ogmios_config)?;
+
+	let offchain = context.offchain_impl(ogmios_config)?;
+	let (payment_key, governance_authority) = get_private_key_and_key_hash(context)?;
+
+	let runtime = tokio::runtime::Runtime::new().map_err(|e| anyhow::anyhow!(e))?;
+	runtime
+		.block_on(offchain.init_governance(governance_authority, payment_key, genesis_utxo))
+		.map_err(|e| anyhow::anyhow!("Offchain call failed: {e:?}!"))?;
```
Comment (LGLO) [inline toolkit/partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs:30]: I try to think like a user. I see: "Offchain call failed: <something>" and now I'm wondering what an "Offchain call" is, but I think in v1.4 we will still have "beta" experience of some CLIs.

**Pull request merged**